src/Component/Application/ApplicationEditForm/ApplicationEditForm.less

```diff
@@ -0,0 +1,18 @@
+.application-table {
```

Comment:
This seems to be the wrong file or class. Not sure if its even used.

Reply:
The whole file can be removed for now, war copy&paste..

Will be deleted in some next followup